apply/apply.go

```diff
@@ -0,0 +1,106 @@
+/* Helper package to merge existing Objects with ones we are trying to create
+ * Shamelessly copied from SRIOV Network Operator
```

Comment:
Could you add the URL you copied from. And also if there are tests there, could you copy those here too?